data/src/main/scala/ch.epfl.scala.index.data/bintray/ListPomsNEw.scala

```diff
@@ -0,0 +1,125 @@
+package ch.epfl.scala.index
+package data
+package bintray
+
+import java.nio.file.Files
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import akka.stream.scaladsl.{FileIO, Flow, Keep, Source}
+import akka.util.ByteString
+import com.github.nscala_time.time.Imports._
+import model._
```

Comment:
I try to order imports by relevance. Like anything from the project at the top, then protocols, transport, utils, etc